lib/buffer.js

```diff
@@ -1259,12 +1259,32 @@ function atob(input) {
   if (arguments.length === 0) {
     throw new ERR_MISSING_ARGS('input');
   }
+
   input = `${input}`;
+  let nonAsciiWhitespaceCharCount = 0;
+
   for (let n = 0; n < input.length; n++) {
-    if (!ArrayPrototypeIncludes(kForgivingBase64AllowedChars,
-                                StringPrototypeCharCodeAt(input, n)))
+    const index = ArrayPrototypeIndexOf(
+      kForgivingBase64AllowedChars,
+      StringPrototypeCharCodeAt(input, n));
+
+    if (index > 4) { // The first five char are ASCII whitespace.
```

Comment:
Probably worth trying to make this comment less ambiguous (sorry my original suggestion was not very good 😬).
```suggestion
    // The first 5 elements of `kForgivingBase64AllowedChars` are
    // ASCII whitespace char codes.
    if (index > 4) {
```